src/composables/useTrackers.ts

```diff
@@ -131,6 +132,12 @@ export function useTrackers(flawUuid: string, affects: Ref<ZodAffectType[]>) {
     }
   }
 
+  function toggleTrackerSelections(affectStreams: UpdateStream[]) {
```

Comment:
function updateTrackerSelections(affectStreams: UpdateStream[]) {
```suggestion
  function updateTrackerSelections(affectStreams: UpdateStream[]) {
```